presto-orc/src/test/java/io/prestosql/orc/BenchmarkStreamReaders.java

```diff
@@ -945,13 +1272,11 @@ private OrcRecordReader createRecordReader()
     {
         protected File temporaryDirectory;
         protected File varcharWithNullFile;
-        private Random random;
 
         @Setup
         public void setup()
                 throws Exception
         {
-            random = new Random(0);
```

Comment:
I think you should use it in `createVarcharValuesWithNulls`

Reply:
But for `createVarcharValuesWithNulls` we create a new `Random` and use it so ideally it is not required

```
        private List<String> createVarcharValuesWithNulls()
        {
            Random random = new Random();
```

Reply:
Yes, but benchmarks should be deterministic and `Random(0)` has seed set explicitly to  `0`

Reply:
Okay so we need to set this for all the `BenchmarkData` right ?

Reply:
yes, in fact I think you can just create `Random` instance locally within `create*` methods.